apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -119,6 +119,10 @@ const BookingPage = ({
         collectPageParameters("/book", { isTeamBooking: document.URL.includes("team/") })
       );
     }
+    localStorage.removeItem("formerTime");
```

Comment:
I am thinking whether it makes sense to still pass the formerTime via a query parameter. I think the localstorage solution is a bit prone to breaking, because if you're able to book an event without this component, the storage wouldn't get cleared, and you would see an incorrect/old "before" time. Perhaps as of right now that's not possible, but in the future it might, and you need to keep the fact that this localstorage items needs to be cleared in your mental model to think about it when building new functionality, in order to not break this. 

If we would turn this into a query param however, you can also forget to pass this param, but in that case it's not broken since we still show the correct new date and time, only the previous date isn't shown. So I would suggest changing this to a query param to be more future proof 🙏